src/server/network.ts

```diff
@@ -401,6 +430,35 @@ export class Response extends SdkObject {
   frame(): frames.Frame {
     return this._request.frame();
   }
+
+  transferSize(): number | undefined {
+    return this._transferSize;
+  }
+
+  bodySize(): number | undefined {
+    if (!this._transferSize)
+      return;
+    return this._transferSize - this.headersSize();
```

Comment:
You are going to see some negative sizes here.

Reply:
In theory transfer size is always bigger, what would you change here?

Reply:
The logic that you use when computing sizes is fuzzy, so you should tolerate inaccurate numbers, capping this to zero. Does transfer size include headers?